### ISSUE TITLE
No Windows binaries for release

### ISSUE THREAD

**Issue by garethr** [kubernetes-sigs/kind#155]
The current release https://github.com/kubernetes-sigs/kind/releases/tag/0.0.1 doesn't have available Windows binaries.

**Comment by BenTheElder** [comment]
yes, I didn't actually get a chance to test code-as-released on windows yet. kind isn't regularly seeing usage on windows at the moment to my knowledge so it's been on the back-burner.

I'll test it out tonight and if there are no issues update the release with windows binaries.

/kind bug
/assign
/priority important-soon

**Comment by garethr** [comment]
@BenTheElder thanks. I'll do my best to give it some usage and report any issues I find.

**Comment by BenTheElder** [comment]
I finally got to some testing post-KubeCon -- it looks like the fake filesystem for the kutomize integration needs a work-around for windows, xref https://github.com/kubernetes-sigs/kind/issues/181

I'll get a fix in and be sure the next release does have windows binaries :-)

We'll have to look at windows + docker CI at some point, not sure what the best option is there. Maybe we can get windows clusters in the k8s CI some day 😉

**Comment by garethr** [comment]
Thanks! @BenTheElder https://www.appveyor.com/ is often used I think.

**Comment by BenTheElder** [comment]
Windows should be usable at HEAD again, though the instructions for `export KUBECONFIG` won't work in cmd.exe of course, will open another issue for that.

I think the next release will wind up being sometime in Q1, as we have some major changes in flight (multi-node!).

We're aiming for a "V1ish" release in terms of forward compatibility by the end of Q1 / the Kubernetes 1.14 release and I definitely want to have windows binaries in the first "GA-ish" release, if not sooner (ideally sooner... whenever we get a release containing the fixes)!
